_delphi_utils_python/delphi_utils/validator/dynamic.py

```diff
@@ -593,6 +596,10 @@ def check_avg_val_vs_reference(self, df_to_test, df_to_reference, checking_date,
         #  - Use to calculate z-score for each test datapoint for a given geo_id and date.
         #  - Avg z-scores over each geo_id, across all dates.
         #  - Avg all z-scores together.
+        num_ref_dates = max(14, self.params.max_check_lookbehind.days)
```

Comment:
To match the suggested change above,

```suggestion
        num_ref_dates = max(7, self.params.max_check_lookbehind.days)
```